src/sentry/web/client_config.py

```diff
@@ -43,7 +43,12 @@
 from sentry.utils.email import is_smtp_enabled
 from sentry.utils.http import is_using_customer_domain
 from sentry.utils.settings import is_self_hosted, should_show_beacon_consent_prompt
-from sentry.utils.support import get_support_mail
+
+
+def _get_support_mail() -> str | None:
+    """Returns the most appropriate support email address"""
+
+    return options.get("system.support-email") or options.get("system.admin-email") or None
```

Comment:
Maybe? 

```suggestion
    return options.get("system.support-email", None) or options.get("system.admin-email", None)
```

Reply:
the default for `get` is already `None`

Reply:
So, the last `None` is not required?

Reply:
it's a little bit different -- in that options have a builtin default -- and that can be empty or None -- the `or None` handles it being set but to a falsey value